modules/juce_graphics/colour/juce_ColourGradient.h

```diff
@@ -37,6 +37,16 @@ namespace juce
 class JUCE_API  ColourGradient  final
 {
 public:
+    //==============================================================================
+    /** The available gradient types. */
+    enum GradientType

```

Comment:
My suggestion here is to use an `enum class`.

In terms of naming conventions - as per the JUCE coding standards:
>For enums, use camel-case with the same capitalisation that you'd use for a class and its member variables, e.g.
```c++
enum class MyEnum
{
    enumValue1 = 0,
    enumValue2 = 1
};
```